src/MallardMessageHandlers/MallardMessageHandlers.csproj

```diff
@@ -9,9 +9,20 @@
 		<PackageId>MallardMessageHandlers</PackageId>
 		<Description>MallardMessageHandlers</Description>
 		<GenerateDocumentationFile>true</GenerateDocumentationFile>
+		<PackageTags>mallardmessagehandlers</PackageTags>
```

Comment:
Not sure people will search for "mallardmessagehandlers" :D
We could use things like DelegatingHandler, oauth, network-status, etc.
Basically have tags based on the features.